mozetl/symbolication/modules_with_missing_symbols.py

```diff
@@ -167,7 +167,10 @@ def are_symbols_available(debug_file, debug_id):
     for name, (version, debug_id, debug_file), count in top_missing
 ]
 
-subject = "Weekly report of modules with missing symbols in crash reports"
+today_date = datetime.today().strftime("%Y-%m-%d")
```

Comment:
This uses `datetime.today()` because that's what the script is querying with. The script takes a `date` param, but that's only used to determine whether the script should run and isn't used for the queries.